src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write your code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const tfoot = document.querySelector('tfoot');
+
+const rowsHead = Array.from(thead.querySelectorAll('tr'));
+const rows = Array.from(tbody.querySelectorAll('tr'));
+const rowsFoot = Array.from(tfoot.querySelectorAll('tr'));
+
+rowsHead.forEach((row) => {
+  const cells = Array.from(row.querySelectorAll('th'));
+
+  const secondCell = cells[1].cloneNode(true);
```

Comment:
Ensure that there are at least two cells in the row before attempting to clone the second cell. Otherwise, this will throw an error if `cells[1]` is undefined.